src/Symfony/Component/Config/Definition/Builder/BooleanNodeDefinition.php

```diff
@@ -31,18 +31,6 @@ public function __construct($name, NodeParentInterface $parent = null)
     }
 
     /**
-     * {@inheritdoc}
-     *
-     * @deprecated Deprecated since version 2.8, to be removed in 3.0.
-     */
-    public function cannotBeEmpty()
```

Comment:
As I understand correctly - after removing this overridion, calling this method just executes from parent class without any error?